### DISCUSSION TITLE
Overwrite the default keymap <C-/> with the plug in keymap | lazyterm to toggleterm

### DISCUSSION THREAD

**Discussion by afzx-ahmad** [LazyVim/LazyVim#2559] [Q&A] [answered]
How can I change/overwrite the <C-/> keymap to use toggleterm ?

this is not work :)
![image](https://github.com/LazyVim/LazyVim/assets/72065890/163003da-e215-42ff-9a91-d8bc1a497ac4)

![image](https://github.com/LazyVim/LazyVim/assets/72065890/2eeffcf3-992a-46ed-9f2e-605794ca7457)

**Comment by dpetka2001** [comment]
```lua
return {
  {
    "akinsho/toggleterm.nvim",
    opts = { direction = "float" },
    keys = {
      { "<C-/>", "<cmd>ToggleTerm<cr>", desc = "Open ToggleTerm" },
    },
  },
}
```
This seems to work for me. No need to delete the keymaps.

**Reply by afzx-ahmad** [reply]
Thank you for responding,
but when i press C-/, nvim keeps opening lazyterm instead of toggleterm
![image](https://github.com/LazyVim/LazyVim/assets/72065890/304d0af8-d177-4bbb-be8b-4b6bc6e224d1)

**Reply by dpetka2001** [reply]
What do you have in your `keymaps.lua`?

**Reply by afzx-ahmad** [reply]
nothing
![image](https://github.com/LazyVim/LazyVim/assets/72065890/7702b2b7-79a1-4640-baca-f3906cf43a99)

**Reply by dpetka2001** [reply]
Enter insert mode in Neovim and press `<C-v><C-/>`. What does it show?

**Reply by afzx-ahmad** [reply]
nothing happened,  ^_^_

**Reply by dpetka2001** [reply]
That's not possible. It should show something. What terminal do you use? Do you use tmux? Take a screenshot with you pressing `<C-v>` and then `<C-/>` in insert mode.

**Reply by dpetka2001** [reply, marked as answer]
Try to set in your keys `<C-_>` instead of `<C-/>`. That's the most common key sequence that some terminals (tmux included) might translate `<C-/>` to.

**Reply by afzx-ahmad** [reply]
yeah, i use tmux, finnaly this is work
![image](https://github.com/LazyVim/LazyVim/assets/72065890/e20b019f-050b-4d78-85c4-c2d3d4ce03c0)

thanks @dpetka2001 you are helpfull

**Reply by DaKeypunchAr** [reply]
I have the same problem, but this solution does not solve it.
